src/items.rs

```diff
@@ -257,25 +279,30 @@ pub fn filter_today_and_has_time(items: Vec<Item>, config: &Config) -> Vec<Item>
 pub fn set_priority(config: &Config, item: items::Item) {
     println!("{}", item.fmt(config, FormatType::Single));
 
-    let options = vec!["1", "2", "3"].iter().map(|s| s.to_string()).collect();
-    let priority =
-        config::select_input("Choose a priority from 1 (lowest) to 3 (highest):", options)
-            .expect("Please enter a number from 1 to 3");
-
-    match priority.as_str() {
-        "1" => {
+    let options = vec![Priority::Low, Priority::Normal, Priority::High];
+    let priority = Select::new(
+        "Choose a priority that should be assigned to task:",
+        options,
+    )
+    .prompt()
+    .map_err(|e| e.to_string())
+    .expect("Failed to create option list of priorities");
+
+    match priority {
+        //TODO maybe create some map to easily convert priority to u8
```

Comment:
Make a ticket?

Reply:
I think I have resolved it, at least for now it looks ok